webapp/src/shared/urls.ts

```diff
@@ -9,6 +9,7 @@ export const EDIT_USER_URL = `${USER_ME_URL}/edit`;
 export const EDIT_AVATAR_URL = `${USER_ME_URL}/edit/avatar`;
 export const PLAY_URL = '/play';
 export const CHAT_URL = '/chat';
+export const CREATE_CHATROOM_URL = '/createchatroom';
```

Comment:
question: What do we think about renaming this endpoint to `chat/create` or `chatroom/create`?

Reply:
I would say `chatroom/create` or `chat/createchatroom`, but I don't mind naming it any other way